tests/unit/BloomFilterSyncTest.cpp

```diff
@@ -0,0 +1,127 @@
+/* This code is part of the GenSync project developed at Boston University.  Please see the README for use and references. */
+//
+// Created by Anish Sinha on 12/4/23.
+//
+
+#include "BloomFilterSyncTest.h"
+#include <GenSync/Syncs/GenSync.h>
+#include <GenSync/Syncs/BloomFilterSync.h>
+#include "../TestAuxiliary.h"
+CPPUNIT_TEST_SUITE_REGISTRATION(BloomFilterSyncTest);
+
+BloomFilterSyncTest::BloomFilterSyncTest() = default;
+
+BloomFilterSyncTest::~BloomFilterSyncTest() = default;
+
+void BloomFilterSyncTest::setUp() {
+	const int SEED = 93;
+	srand(SEED);
+}
+
+void BloomFilterSyncTest::tearDown() {
+}
+
+void BloomFilterSyncTest::BloomFilterSyncSetReconcileTest() {
+	const int BITS = sizeof(randZZ());
+	
+	GenSync GenSyncServer = GenSync::Builder().
+			setProtocol(GenSync::SyncProtocol::BloomFilterSync).
+			setComm(GenSync::SyncComm::socket).
+			setBits(BITS).
+			setExpNumElems(numExpElem).
+			setFalsePosProb(0.001).
+			build();
+	
+	GenSync GenSyncClient = GenSync::Builder().
```

Comment:
Are you actually putting any elements into the filters?

Reply:
I believe the syncTest method handles putting elements into the filters. I followed the other tests when implementing this and it seems as though all of them follow this same implementation.

Reply:
ok